app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, counter_cache: :comments_counter
+
+  def update_counters
+    update(comments_counter: comments.count)
+  end
```

Comment:
- [ ] I think it would be better if you update your ` update_counters` method of comments to match the required result. According to this **[article](https://www.geeksforgeeks.org/ruby-string-count-method/)** the `increment` method will be good than the `count` method. As you are supposed to count the newly added comments. **[Here](https://stackoverflow.com/questions/11499110/increment-vs-1)** is an example of how to use it